python/README.md

```diff
@@ -46,19 +46,15 @@ Runs on both Windows and Linux.
 
 ### How does endpoint discovery work?
 
-The mechanism to discover the CloudControl IP address is to resend a DHCP request. In the centralized DHCP service, the response is customized 
-by carrying the CloudControl IP address inside the DHCP response option 245. The endpoint discovery script uses the same mechanism to determine 
-the CloudControl IP address. Once the IP address is obtained, it is made available as part of the environment variable both in Windows and Linux worlds.
-
+The mechanism to discover the ScheduledEvents IP address is to resend a DHCP request. In the centralized DHCP service, the response is customized by carrying the ScheduledEvents IP address inside the DHCP response option 245. The endpoint discovery script uses the same mechanism to determine the ScheduledEvents IP address. Once the IP address is obtained, it is made available as part of the environment variable both in Windows and Linux worlds.
 
 ### Script usage
 
-By default, upon discovering the CloudControl IP address, the script adds it to the environment variables. The variable name in both Windows and 
-Linux is “CLOUDCONTROLIP”. However, the script has two additional modes of operation – “--debug” and “--donotaddtoenv” as listed below.
+By default, upon discovering the ScheduledEvents IP address, the script adds it to the environment variables. The variable name in both Windows and Linux is “SCHEDULEDEVENTSIP”. However, the script has two additional modes of operation – “--debug” and “--donotaddtoenv” as listed below.
```

Comment:
This is kind of a lie - it has three arguments! You can also opt not to output to registry

I think this will also only work on windows, due to the way the import is setup. Likely want to test + refactor

Reply:
I will be re-sending the PR again once Mauricio has submitted his changes to unblock him and avoid having to resolve conflicts multiple times. Will be taking your feedback into account at that time.